### PR TITLE
Update GraphiQL

### PR DESCRIPTION
**Pull request by leebyron**
Also, copy the new example GraphiQL HTML to enable link sending

### CODE REVIEW

File: parse/src/public/graphiql/index.html [hunk 1]
```diff
@@ -16,6 +16,58 @@
...
+        }
+      });
+
+      // if variables was provided, try to format it.
+      if (parameters.variables) {
+        try {
+          parameters.variables =
+            JSON.stringify(JSON.parse(query.variables), null, 2);
+        } catch (e) {
+          // Do nothing
```
Comment (schrockn) [inline parse/src/public/graphiql/index.html]: don't want to display error of some sort?

**Comment by schrockn** [comment]
looks good to me other than q about the error

File: parse/src/public/graphiql/index.html (hunk 1 above)
Comment (leebyron) [inline parse/src/public/graphiql/index.html]: Nah, this happens on every keystroke, so 90% of them are likely to be invalid JSON

Comment (leebyron) [inline parse/src/public/graphiql/index.html]: Ah wait, this is the parse side of it - so not per keystroke, but refreshing a page while you're in the middle of typing JSON would result in incomplete JSON here - so similar point.

Comment (schrockn) [inline parse/src/public/graphiql/index.html]: makes sense. leave a comment and engage.

Comment (leebyron) [inline parse/src/public/graphiql/index.html]: https://github.com/graphql/graphiql/commit/50d5e2ae78960259c19ecf37216e65f0ced66916

**Pull request merged**